### DISCUSSION TITLE
editing metadata in epub without recreating the file.

### DISCUSSION THREAD

**Discussion by dbojan** [jgm/pandoc#10406] [Q&A]
Hi.

-Is it possible to edit metadata in epub without recreating the file?
For example to change title to something else.

-if so, is it possible to use other metadata, for example to change
title
to
series + title

Thanks.

**Comment by badumont** [comment]
It should be possible with specialized software such as Calibre (https://calibre-ebook.com/). By design, Pandoc cannot be used to modify a file without recreating it.
